make_queue/forms.py

```diff
@@ -153,3 +154,32 @@ class FreeSlotForm(forms.Form):
         initial=1, label=_("Machine type"))
     hours = IntegerField(min_value=0, initial=1, label=_("Duration in hours"))
     minutes = IntegerField(min_value=0, max_value=59, initial=0, label=_("Duration in minutes"))
+
+
+class BaseMachineForm(forms.ModelForm):
+    class Meta:
+        model = Machine
+        fields = "__all__"
+        widgets = {
+            "status": SemanticChoiceInput(),
+            "machine_type": SemanticChoiceInput(),
+            "location": MazemapSearchInput(url_field="location_url"),
+        }
+
+
+class CreateMachineForm(BaseMachineForm):
+    class Meta(BaseMachineForm.Meta):
+        exclude = ["status"]
+
+
+class EditMachineForm(BaseMachineForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["status"] = forms.ChoiceField(choices=(
+            ("F", _("Available")),
+            ("O", _("Out of order")),
+            ("M", _("Maintenance")),
+        ))
```

Comment:
This does not seem to work

Reply:
It works for me. I only get these three options and can create and update machines.
![image](https://user-images.githubusercontent.com/32880010/53197368-584cf980-361a-11e9-99f0-2f58b69421c5.png)

What happens at your machine?

Reply:
It works now. I was probably just a commit behind or something.